dispatcher/brokers/pg_notify.py

```diff
@@ -33,6 +36,7 @@ def create_connection(**config) -> psycopg.Connection:
 
 class Broker:
     NOTIFY_QUERY_TEMPLATE = 'SELECT pg_notify(%s, %s);'
+    SELF_CHECK_CHANNEL = 'pgnotify_self_check'
```

Comment:
This would result in all nodes sending their health check messages to all connected nodes. This isn't a good idea for scaling, but also wouldn't work unless there was some other identifying information in the message it used to verify that it came from itself. So this could be addressed by adding a uuid to it, and adding some expected uuid to the message might be a good idea anyway to rule out any possibility of getting wires crossed.